src/ArborX_HDBSCAN.hpp

```diff
@@ -0,0 +1,54 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+#ifndef ARBORX_HDBSCAN_HPP
+#define ARBORX_HDBSCAN_HPP
+
+#include <ArborX_Dendrogram.hpp>
+#include <ArborX_MinimumSpanningTree.hpp>
+
+#include <Kokkos_Profiling_ProfileSection.hpp>
+
+#include <iomanip>
+#include <limits>
+
+namespace ArborX::Experimental
+{
+
+template <typename ExecutionSpace, typename Primitives>
+auto hdbscan(ExecutionSpace const &exec_space, Primitives const &primitives,
+             int core_min_size)
```

Comment:
Should the last argument have a default value?

Reply:
I don't think so, I'm not sure if there is a reasonable default.